Vagrantfile

```diff
@@ -65,14 +65,18 @@ Vagrant.configure(2) do |config|
     env: {"AWS_ACCESS_KEY_ID": ENV["AWS_ACCESS_KEY_ID"],
           "AWS_ACCESS_SECRET": ENV["AWS_ACCESS_SECRET"],
           "AWS_REGION": ENV["AWS_REGION"],
-          "UPLOAD_SERVICE_SENTRY_DSN": ENV["UPLOAD_SERVICE_SENTRY_DSN"],
+          "APP_USER": "vagrant",
+          "DATABASE_URL": "postgres://vagrant:localdb@127.0.0.1:5432/permanent?sslmode=disable",
```

Comment:
I agree it would be best to put this in the `.env`, and there's nothing sensitive in this string so it should be fine to include the whole thing in `.env.template`.

Reply:
Done!